### PR TITLE
T 15309 relay meter dockerhub

### PR DESCRIPTION
**Pull request by rutwickjain87**
This change introduces a new Github workflow which will build, tag and push docker images for - 
1) relay meter collector
2) relay meter apiserver

They will have their own Dockerhub repo to maintain images separately.

### CODE REVIEW

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'v*.*.*'
```
Comment (WesleyCharlesBlake) [inline .github/workflows/dockerhub.yml:7]: lets add the tag event as well:

```
    tags:
      - 'v*'
```

**Review by WesleyCharlesBlake — DISMISSED** [review]
LGTM @rutwickjain87 , lets just add the tag event, and then you can merge when you're ready

File: .github/workflows/dockerhub.yml (hunk 1 above)
Comment (rutwickjain87) [reply .github/workflows/dockerhub.yml:7]: done

**Review by WesleyCharlesBlake — APPROVED** [review]
LGMT

**Pull request merged**